README.md

```diff
@@ -1,34 +1,27 @@
-This README is just a fast *quick start* document. You can find more detailed documentation at [redis.io](https://redis.io).
+This README is just a fast *quick start* document. You can find more detailed documentation at [valkey.io](https://valkey.io/)
 
-What is Redis?
---------------
-
-Redis is often referred to as a *data structures* server. What this means is that Redis provides access to mutable data structures via a set of commands, which are sent using a *server-client* model with TCP sockets and a simple protocol. So different processes can query and modify the same data structures in a shared way.
-
-Data structures implemented into Redis have a few special properties:
+What is Valkey?
+---------------
 
-* Redis cares to store them on disk, even if they are always served and modified into the server memory. This means that Redis is fast, but that it is also non-volatile.
-* The implementation of data structures emphasizes memory efficiency, so data structures inside Redis will likely use less memory compared to the same data structure modelled using a high-level programming language.
-* Redis offers a number of features that are natural to find in a database, like replication, tunable levels of durability, clustering, and high availability.
+Valkey is an open source fork of Redis OSS, created at the point where Redis switched to a proprietary license.
 
-Another good example is to think of Redis as a more complex version of memcached, where the operations are not just SETs and GETs, but operations that work with complex data types like Lists, Sets, ordered data structures, and so forth.
+Valkey is often referred to as a *data structures* server. What this means is that Valkey provides access to mutable data structures via a set of commands, which are sent using a *server-client* model with TCP sockets and a simple protocol. So different processes can query and modify the same data structures in a shared way.
 
-If you want to know more, this is a list of selected starting points:
+Data structures implemented into Valkey have a few special properties:
 
-* Introduction to Redis data types. https://redis.io/topics/data-types-intro
-* Try Redis directly inside your browser. https://try.redis.io
-* The full list of Redis commands. https://redis.io/commands
-* There is much more inside the official Redis documentation. https://redis.io/documentation
+* Valkey cares to store them on disk, even if they are always served and modified into the server memory. This means that Valkey is fast, but that it is also non-volatile.
```

Comment:
```suggestion
* Valkey can be configured store copies of the data on disk, even if they are always served and modified in memory. This means that Valkey is fast, but can also durably store the data.
```

Reply:
I didn't write this text. Just changed redis to valkey. (It's probably antirez wordings, which I quite like.)

Reply:
Where is the wording from?

Reply:
What i did is i just edited this file on the 7.2 branch. I didn't compare to what's on unstable.